casepro/backend/tests/test_rapidpro.py

```diff
@@ -862,14 +866,34 @@ def test_push_outgoing(self, mock_create_broadcast, mock_send_raw_email):
         )
         mock_create_broadcast.reset_mock()
 
-    def test_push_contact(self):
+    @patch("dash.orgs.models.TembaClient.get_contacts")
+    @patch("dash.orgs.models.TembaClient.create_contact")
+    def test_push_contact(self, mock_create_contact, mock_get_contacts):
         """
-        Pushing a new contact should be a noop.
+        If a contact is added in CasePro,
+        it should be added in RapidPro and the uuid should match
```

Comment:
I think it's best to remove the mention of the uuid matching, since your test doesn't cover that